js_modules/dagster-ui/packages/ui-core/src/app/App.tsx

```diff
@@ -47,3 +55,8 @@ const Container = styled.div`
   display: flex;
   height: calc(100% - 64px);
 `;
+
+const ChildContainer = styled.div`
+  height: 100%;
+  overflow: hidden;
+`;
```

Comment:
Does this need overflow scroll? or is Main the scroller?

Reply:
`MainContent` is the scrollable child, as defined in `ContentRoot`. It's passed in as `children` here.